docs/running.md

```diff
@@ -18,6 +18,9 @@ sudo docker run \
 
 cAdvisor is now running (in the background) on `http://localhost:8080/`. The setup includes directories with Docker state cAdvisor needs to observe.
 
+**Note**: If you're running docker daemon with user namespace enabled, the above command will run into
```

Comment:
Nits

-the above command will emit the following error ...
-move to avoid it... to a new sentence